### ISSUE TITLE
<use [...] /> is cut off

### ISSUE THREAD

**Issue by georg94** [ariutta/svg-pan-zoom#52]
When I use symbols and uses to decrease the size of an svg because I use the same element multiple times I get some issues with svg-pan-zoom:

``` svg
<?xml version="1.0" encoding="utf-8"?>
<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns="http://www.w3.org/2000/svg">
<defs>
  <symbol id="my_id">
    <circle r="10" />
  </symbol>
</defs>
<g>
  <use xlink:href="#my_id"/>
</g>
<g transform="scale(1.5)">
  <use xlink:href="#my_id" x="2000" y="1000" />
</g>
```

When I use this svg on a screen that has not a height of 100px and a width of 2000px the second element is not visible.
In my actual project I have paths that are displayed until the screen ends, but when I pan the svg up the path is cut off.

**Comment by gallagherrchris** [comment]
This seems to be an issue related to viewboxing.  By removing the viewbox on the parent and applying on to the symbol, I was able to make the full spiral appear.

http://jsfiddle.net/cf4Sa/2/

In the MDN examples given, viewboxes are used and x,y,height,width are applied on the use elements. https://developer.mozilla.org/en-US/docs/Web/SVG/Element/symbol

**Comment by bumbu** [comment]
@gallagherrchris Nice one! 
So a solution would be to apply viewbox to `use` elements in case that they don't have one? Do you know if it is possible to get the sizes of an `use` element so to be able to set the right `viewBox`?

**Comment by gallagherrchris** [comment]
@bumbu No, the use element doesn't need anything extra on it.  The symbol contains the viewbox which will describe what size you want that symbol to be and the use element will respect tat (not too sure on how it works in the spec if it's and implicit inherit or not).  In the modified fiddle, you can see that the use element has no attributes on it aside from the link.

Now it does appear to be possible to calculate the viewbox of the path element on the fly using js.  If you have a framework you should be able to getBBox and use that.  If you don't have a framework, you can parse the values from the 'd' attribute on the path and calculate the max and min and use viewbox="min min max max"
